src/battle_ai_main.c

```diff
@@ -174,7 +174,7 @@ static u32 GetAiFlags(u16 trainerId)
             flags = GetTrainerAIFlagsFromId(trainerId);
     }
 
-    if (gBattleTypeFlags & BATTLE_TYPE_DOUBLE)
+    if (BATTLE_TYPE_IS_DOUBLE)
```

Comment:
what is the reason for the macro? 

Unless you want to use a macro for every battle type which I don't see a reason for it I would revert it

Reply:
oh, there is actually a macro for it already and a function. Since it already exists could you use it instead? `IsDoubleBattle()'

Actually I would do this:

Add
```
static inline bool32 IsDoubleBattle(void)
{
    return gBattleTypeFlags & BATTLE_TYPE_DOUBLE;
}
```
in `include/battle.h` at the bottom and remove `IsDoubleBattle` and `IS_DOUBLE_BATTLE` from `battle_anim_mons.c`

Reply:
https://github.com/rh-hideout/pokeemerald-expansion/pull/5131/commits/b1947b840c584f1d0a67d76fb6d227cce23946ef